### PR TITLE
Update examples/glusterfs to v1beta3; add desciption of Glusterfs endpoints in README.md

### PR DESCRIPTION
**Pull request by caesarxuchao**
Update examples/glusterfs to v1beta3; add desciption of Glusterfs endpoints in README.md.

@rootfs @nikhiljindal

### CODE REVIEW

**Comment by caesarxuchao** [comment]
fixes #8214

**Comment by rootfs** [comment]
thanks for the v1beta3 endpoints example.

File: examples/glusterfs/README.md [hunk 1]
```diff
@@ -2,15 +2,45 @@
...
+
+```
+The "IP" field should be filled with the address of a node in the Glusterfs server cluster. The "port" field is not really used, so any value between 1 and 65535 will work.
+
+Create the endpoints,
+```shell
+$ kubectl create -f examples/glusterfs/glusterfs-endpoints.json
+```
+
+You can verify that the endpoints are succussfully created by running
```
Comment (nikhiljindal) [inline examples/glusterfs/README.md]: typo in successfully

**Comment by nikhiljindal** [comment]
LGTM % a minor typo.

**Comment by caesarxuchao** [comment]
Fixed. Thank you @nikhiljindal.

File: examples/glusterfs/README.md [hunk 2]
```diff
@@ -2,15 +2,45 @@
...
+      ],
+      "ports": [
+        {
+          "port": 1,
+          "protocol": "TCP"
+        }
+      ]
+
+```
+The "IP" field should be filled with the address of a node in the Glusterfs server cluster. The "port" field is not really used, so any value between 1 and 65535 will work.
```
Comment (nikhiljindal) [inline examples/glusterfs/README.md]: Can we skip the field if its not used?

Comment (caesarxuchao) [inline examples/glusterfs/README.md]: No. If we skip "ports", then endpoints will not be created.

Comment (nikhiljindal) [inline examples/glusterfs/README.md]: Then, the statement that "port field is not really used" is not true.
I will suggest changing it to say something like: "In this example, it is fine to give any valid value (from 1 to 65535) to the port field"

Comment (caesarxuchao) [inline examples/glusterfs/README.md]: Good point! Thanks.

**Comment by caesarxuchao** [comment]
Updated. Thanks. @nikhiljindal

**Comment by nikhiljindal** [comment]
LGTM, thanks!

**Comment by nikhiljindal** [comment]
Shippable is green. Merging

**Pull request merged**